catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/managers/AccessManagerImpl.java

```diff
@@ -160,11 +158,15 @@ public int get_timeout(Request aRequest)
 
   @Override
   public void set_timeout(Request aRequest, int new_lifetime)
-      throws InvalidInputParameter, ProcessingFault, SystemFault {}
+      throws InvalidInputParameter, ProcessingFault, SystemFault {
+    // This method is not expected to be called
```

Comment:
@troymohl would it be more appropriate to throw `NO_IMPLEMENT` here instead of a comment?

https://github.com/mcalcote/alliance/blob/master/catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java#L310

Reply:
I don't recall off the top of my head, but I think some of these empty methods don't throw exceptions because there was something calling those methods that couldn't handle a NO_IMPLEMENT exception.

Reply:
Will leave with just a comment for now 👍